docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,62 @@
+# Release 460 (3 Oct 2024)
+
+## General
+
+* Fix query failures when complex symbols are referenced from within lambda expression. ({issue}`23649`)
```

Comment:
This needs to be reworded. "complex symbols" isn't something users understand. Maybe just say:

"Fix failure for certain queries involving lambda expressions".

Reply:
Done